helm-chart/eoapi/templates/services/nginx-doc-server.yaml

```
@@ -46,6 +46,14 @@ spec:
       - name: doc-html-{{ .Release.Name }}
         configMap:
           name: nginx-root-html-{{ .Release.Name }}
+      {{- with .Values.docServer.settings.affinity }}
+      affinity:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      {{- with .Values.docServer.settings.tolerations }}
+      tolerations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

Can we check for the existence of `affinity` and `tolerations`, and possibly avoid adding them as empty objects to `values.yaml`?

```suggestion
        {{- if .Values.docServer.settings }}
        {{- if .Values.docServer.settings.affinity }}
        affinity:
          {{- toYaml .Values.docServer.settings.affinity | nindent 8 }}
        {{- end }}
        {{- if .Values.docServer.settings.tolerations }}
        tolerations:
          {{- toYaml .Values.docServer.settings.tolerations | nindent 8 }}
        {{- end }}
        {{- end }}
```